tools/import_grs/import_grs.py

```diff
@@ -0,0 +1,250 @@
+import datetime
+import errno
+import logging
+import os
+import re
+import sys
+import yaml
+from xlrd import open_workbook
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
```

Comment:
@Redbeard0220 Should this be `sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))`?  @mrinalirao and I both installed the `safetyculture-sdk-python` via pip, which doesn't contain the new grs import feature (`get_response_sets` not defined in the class), as a result, when running the script we got `AttributeError: SafetyCulture instance has no attribute 'get_response_sets'`